src/evaluator.rs

```diff
@@ -57,18 +62,18 @@ impl<'a> Evaluator {
 				Atom::Int(i) => Atom::Int(i),
 				Atom::Symbol(_) => {
 					return Err(LockjawRuntimeError::InvalidArguments(
-						"Expected number got Symbol",
+						"Expected number got Symbol".to_string(),
 					))
 				}
 			},
 			Expression::SExpression(_) => {
 				return Err(LockjawRuntimeError::InvalidArguments(
-					"Expected number got SExpr",
+					"Expected number got SExpr".to_string(),
 				))
 			}
 			Expression::QExpression(_) => {
 				return Err(LockjawRuntimeError::InvalidArguments(
-					"Expected number got SExpr",
+					"Expected number got SExpr".to_string(),
```

Comment:
these cases seem like they could be consolidated with a fall through case if you add  the display trait (or some similar printing method) for `Expression`